src/CyBy/Draw/Internal/Graph.idr

```diff
@@ -465,7 +465,7 @@ selectionCorners (G o g) =
 
 ||| Checks, if there is enough space to grab the box in the canvas.
 export
-selectZones : (s : CoreDims) => (p1,p2 : Point Id) -> SelectZones
+selectZones : {auto s : CoreDims} -> (p1,p2 : Point Id) -> SelectZones
```

Comment:
Please undo this. The style guide uses `{auto ...}` for *multiline* argument lists